libimage/save_test.go

```diff
@@ -69,8 +80,13 @@ func TestSave(t *testing.T) {
 		// First clean up all images and load the cache.
 		_, rmErrors := runtime.RemoveImages(ctx, nil, nil)
 		require.Nil(t, rmErrors)
-		_, err = runtime.Load(ctx, imageCache.Name(), loadOptions)
-		require.NoError(t, err)
+		if strings.Contains(test.format, "oci-archive") {
```

Comment:
Just to be extra sure :^)
```suggestion
		if test.format == "oci-archive" {
```

Reply:
Fixed